File: jug-cfp-server/jug-cfp-micro/src/main/java/org/jugvale/cfp/model/Evento.java
package org.jugvale.cfp.model;

import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;


import io.quarkus.hibernate.orm.panache.PanacheEntity;


@Entity
public class Evento extends PanacheEntity {

	@Column
	public String nome;

	@Column(length=1000)
	public String descricao;

	@Temporal(TemporalType.TIMESTAMP)
	public Date dataInicio;

	@Temporal(TemporalType.TIMESTAMP)
	public Date dataFim;

	@Column
	public String local;

	@Column
	public String url;

	@Column
	public boolean aceitandoTrabalhos;
	
	@Column
	public boolean inscricoesAbertas;	
	
}

File: jug-cfp-client/src/app/model/Evento.ts
class Evento {

  id: Number;
  
	nome:String;

	descricao:String;

	dataInicio:Date;

	dataFim:Date;

	local:String;

	url: String;

	aceitandoTrabalhos: Boolean;

  inscricoesAbertas:Boolean;

  constructor(id: Number,
              nome:String, 
              descricao:String, 
              dataInicio:Date, 
              dataFim:Date, 
              local:String, 
              url:String, 
              aceitandoTrabalhos:Boolean,
              inscricoesAbertas:Boolean) {
    this.id = id;
    this.nome = nome;
    this.descricao = descricao;
    this.dataInicio = dataInicio;
    this.dataFim = dataFim;
    this.local = local;
    this.url = url;
    this.aceitandoTrabalhos = aceitandoTrabalhos;
    this.inscricoesAbertas = inscricoesAbertas;
  }
}

export { Evento } ;